sensu/plugins/metrics-process-usage.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python
+#
+#  metrics-process-usage.py
+#
+# PLATFORMS:
+#   Linux
+#
+# DEPENDENCIES:
+#  Python 2.7+ (untested on Python3, should work though)
+#  Python module: psutil https://pypi.python.org/pypi/psutil
+#
+# USAGE:
+#
+#  metrics-process-usage.py -n <process_name> -w <cpu_warning_pct> -c <cpu_critical_pct> -W <mem_warning_pct> -C <mem_critical_pct> [-s <graphite_scheme>]
+#
+# DESCRIPTION:
+# Finds the pid[s] corresponding to a process name and obtains the necessary
+# cpu and memory usage stats. Returns WARNING or CRITICAL when these stats
+# exceed user specified limits.
+#
+# Code adapted from Jaime Gogo's script in the Sensu Plugins community:
+# https://github.com/sensu-plugins/sensu-plugins-process-checks/blob/master/bin/metrics-per-process.py
```

Comment:
Well, I'm not a lawyer. But this is a derivative work from an MIT licensed file. Presumably this file should also be MIT licensed.